frontend/src/components/Recipes/Content/RecipesPagination.tsx

```diff
@@ -28,7 +41,7 @@ export default function RecipesPagination() {
 
     return (
         <nav className="flex items-center justify-center gap-2">
-            <Link to={`/recipes?_page=${page - 1}`}>
+            <Link to={`/recipes?_page=${page - 1}&${queryParamsString}`}>
```

Comment:
Only if you can't stand the final “&” when there are no filters enabled, you can add it conditionally by checking the length of the queryParamsString.
On the other hand, if you can stand it you can leave it as is 👯

Reply:
Resolved by adding the check and the & directly on the function that returns the queryParamsString